spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/AwsSecretsManagerEnvironmentProperties.java

```diff
@@ -54,6 +54,11 @@ public class AwsSecretsManagerEnvironmentProperties implements EnvironmentReposi
 	 */
 	private String defaultLabel;
 
+	/**
+	 * Do not set staging label when fetching the secret values.
+	 */
+	private boolean ignoreLabel;
```

Comment:
The default value should be `false` in order to maintain the default behavior

Reply:
Primitive boolean is false by default.